pkg/ingester/ingester.go

```diff
@@ -107,6 +121,19 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 
 	f.BoolVar(&cfg.RejectOldSamples, "ingester.reject-old-samples", false, "Reject old samples.")
 	f.DurationVar(&cfg.RejectOldSamplesMaxAge, "ingester.reject-old-samples.max-age", 14*24*time.Hour, "Maximum accepted sample age before rejecting.")
+
+	f.IntVar(&cfg.MaxLengthLabelName, "ingester.max-length-label-name", 1024, "Maximum length accepted for label names")
+	f.IntVar(&cfg.MaxLengthLabelValue, "ingester.max-length-label-value", 2048, "Maximum length accepted for label value. This setting also applies to the metric name")
+
+	hostname, err := os.Hostname()
+	if err != nil {
+		level.Error(util.Logger).Log("msg", "failed to get hostname", "err", err)
+		os.Exit(1)
+	}
+
+	f.StringVar(&cfg.infName, "ingester.interface", "eth0", "Name of network interface to read address from.")
+	f.StringVar(&cfg.addr, "ingester.addr", "", "IP address to register into consul.")
+	f.StringVar(&cfg.id, "ingester.id", hostname, "ID to register into consul.")
```

Comment:
Ditto bad merge / rebase?

Reply:
certainly does... I'll try tidy this today